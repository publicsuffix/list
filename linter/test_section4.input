// test:
// - ICANN section improperly closed

// ===BEGIN ICANN DOMAINS===

example.com

// ===END PRIVATE DOMAINS===
