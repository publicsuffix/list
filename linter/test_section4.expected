8: error: Unexpected end of section: '// ===END PRIVATE DOMAINS==='
8: error: ICANN section not closed
8: warning: No PRIVATE section found
